armi/reactor/blocks.py

```diff
@@ -206,13 +206,18 @@ def makeName(self, assemNum, axialIndex):
 
         This also sets the block-level assembly-num param.
 
+        Once, we used a axial-character suffix to represent the axial
+        index, but this is inherently limited so we switched to a numerical
+        name. The axial suffix needs can be brought in in plugins that require
+        them.
+
         Examples
         --------
         >>> makeName(120, 5)
-        'B0120E'
+        'B0120-005'
         """
         self.p.assemNum = assemNum
-        return "B{0:04d}{1}".format(assemNum, grids.AXIAL_CHARS[axialIndex])
+        return "B{0:04d}-{1:03d}".format(assemNum, axialIndex)
```

Comment:
What clients does `grids.AXIAL_CHARS` have left? Should/can we [re]move it?

Reply:
are we not worried about the asymmetry then? if we allow more blocks in an assem than we have axial chars, then the location labels are going to break in the same way, right?

Reply:
you know what I am actually worried about that! great point. This will fail as soon as a location label is printed out.